graphql/validate.lua

```diff
@@ -262,6 +410,44 @@ local visitors = {
     rules = { rules.uniqueInputObjectFields }
   },
 
+  -- <inputObject>
+  --
+  -- {
+  --   kind = 'inputObject',
+  --   values = list of {
+  --     name = <string>,
+  --     value = <value>,
+  --   },
+  -- }
+  --[[
+  inputObject = {
+  },
+  ]]--
+
+  -- <variable>
+  --
+  -- {
+  --   kind = 'variable',
+  --   name = {
+  --     kind = 'name',
+  --     value = <string>,
+  --   },
+  -- }
+  --[[
+  variable = {
```

Comment:
Here as well

Reply:
In the AST we have only variable name here. It looks pretty simple and, to be honest, I don't know, which kind of example would add something there. If you have an idea, please, let me know.